hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystem.java

```diff
@@ -272,6 +283,15 @@ public void registerListener(Listener listener1) {
     listener = listener1;
   }
 
+  private URI changePrefixFromDfsToBlob(URI uri) throws InvalidUriException {
+    try {
+      String uriString = uri.toString().replace(ABFS_DNS_PREFIX, WASB_DNS_PREFIX);
```

Comment:
ABFS_DNS_PREFIX contains "dfs". should we have ".dfs."
similarly for WASB_DNS_PREFIX.

issue is some uri has dfs in the name apart from actual prefix.